magento/magento2
# Include Reply-To name in contact form email header

### Description
Related to https://github.com/magento/magento2/pull/8654

Current Magento2 code only includes the Reply-To email address, whereas it is useful to also have the Reply-To name, which will always be included due to the Contact Controller validation.

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios
Send an email using Contact Form and check headers include both Email Address and Name of the Contact who sent the email.

### Contribution checklist
